test/extended/testdata/cmd/test/cmd/secrets.sh

```diff
@@ -150,9 +150,3 @@ os::cmd::expect_success 'oc secret unlink --help'
 
 echo "secrets: ok"
 os::test::junit::declare_suite_end
-
```

Comment:
These are being deprecated in https://github.com/openshift/oc/pull/1126 and also don't make sense after we land k8s 1.24, so it's better to get rid of them entirely.